lib/utility/CMakeLists.txt

```diff
@@ -2,9 +2,10 @@ add_library(Logger STATIC)
 target_sources(Logger PRIVATE src/Logger.cpp)
 target_include_directories(Logger PUBLIC include)
 target_set_mbed_dependency(Logger)
-
+add_library(WatchdogWrapper STATIC)
+target_sources(WatchdogWrapper PRIVATE src/WatchdogWrapper.cpp)
+target_include_directories(WatchdogWrapper PUBLIC include)
+target_link_libraries(WatchdogWrapper PUBLIC Logger)
+target_set_mbed_dependency(WatchdogWrapper)
 add_library(LookupTable INTERFACE)
-target_include_directories(LookupTable INTERFACE include)
-
-add_library(units INTERFACE)
-target_include_directories(units INTERFACE include)
```

Comment:
You accidently deleted this part that's adding the units library, that's why the test-units build is failing

Reply:
Not sure why i would have done that, I'll change it for the next commit